superset/assets/src/dashboard/actions/dashboardState.js

```diff
@@ -162,8 +162,7 @@ export function saveDashboardRequestSuccess() {
 
 export function saveDashboardRequest(data, id, saveType, sucessMessage = 'This dashboard was saved successfully.') {
   const path = saveType === SAVE_TYPE_OVERWRITE ? 'save_dash' : 'copy_dash';
-
-  return dispatch =>
+  return dispatch => {
     SupersetClient.post({
```

Comment:
unwanted  addition of  extra { }

Reply:
Both are same....it is not required to remove again {}

Reply:
k